<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dpage</class>
 <widget class="QWidget" name="dpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>标签尺寸设置(mm)：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>20</width>
     <height>541</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="page_widget" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>191</width>
     <height>311</height>
    </rect>
   </property>
   <widget class="CustomLabel" name="label_bar">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>223</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>265</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_ck1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>文本1</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_ck2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>52</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>文本2</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_ck4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>137</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>文本4</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_ck3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>94</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>文本3</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CustomLabel" name="label_ck5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>180</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>文本5</string>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>宽：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="W_lineEdit">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>高：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="H_lineEdit">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>70</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>选择打印属性：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>131</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="PM_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>品名</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ZYPL_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>主营品类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="HH_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>货号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PP_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>品牌</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SLMC_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>四类名称</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="DL_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>大类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ZL_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>中类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="XL_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>小类</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="XB_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="YS_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>颜色</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="GYS_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>供应商</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ZTJ_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>正特价</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="JJ_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>季节</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CGY_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>采购员</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="NLD_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>年龄段</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="NF_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>年份</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CCZ_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>尺寸组</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CGXZ_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>采购性质</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="BX_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>版型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CKG_checkBox">
        <property name="styleSheet">
         <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
        </property>
        <property name="text">
         <string>长宽高</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>95</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="LXSH_checkBox">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>联系商户</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CM_checkBox">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>尺码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="JYFS_checkBox">
      <property name="styleSheet">
       <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
      </property>
      <property name="text">
       <string>经营方式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>是否打印条形码：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BAR_checkBox">
    <property name="styleSheet">
     <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
    </property>
    <property name="text">
     <string>条形码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="styleSheet">
     <string notr="true">font: 20px &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>是否打印序列号：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DATA_checkBox">
    <property name="styleSheet">
     <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
    </property>
    <property name="text">
     <string>序列号</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="QR_pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>使用当前模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QX_pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MR_pushButton">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>490</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BC_pushButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>490</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ZR_pushButton">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>490</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
   </property>
   <property name="text">
    <string>载入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>文本属性设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;楷体&quot;;</string>
   </property>
   <property name="text">
    <string>添加文本框属性设置(mm)：</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>20</width>
     <height>141</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QComboBox" name="font_comboBox">
    <property name="styleSheet">
     <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>宋体</string>
    </property>
    <item>
     <property name="text">
      <string>宋体</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黑体</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>楷体</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="size_comboBox">
    <property name="styleSheet">
     <string notr="true">font: 25 16px &quot;Adobe 宋体 Std L&quot;;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>16</string>
    </property>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Align_comboBox">
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>左对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右对齐</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>文本框间距：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="boxd_lineEdit">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>左间距:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>宽:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="ld_lineEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="wd_lineEdit">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>上间距:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>高:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="td_lineEdit">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hd_lineEdit">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header location="global">customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
